src/actions/uiActions.js

```diff
@@ -118,6 +119,32 @@ export function updateRegistrationAllowed(search) {
 	}
 }
 
+export function updateCompanyData(newData) {
+	return async function(dispatch, getState) {
+		try {
+			const companyData = selectCompanyData(getState())
+			const newCompanyData = { ...companyData, ...newData }
+			updateUi('companyData', newCompanyData)(dispatch)
+		} catch (err) {
+			console.error('ERR_UPDATING_COMPANY_DATA', err)
+			addToast({
+				type: 'cancel',
+				label: translate('ERR_UPDATING_COMPANY_DATA'),
+				timeout: 20000,
+			})(dispatch)
+		}
+	}
+}
+
+export function updateSelectedItems(selectedItems) {
```

Comment:
`selected items` is not very clear naming, if I didn't know this is for receipts I'll never guess.
If you want to use this reducer for anything but the receipts, you should add additional param for id or what items are selected.
```
initialState = {
...,
selectedItems: {}
}
```